packages/amazonq/.changes/next-release/Feature-fe345557-1053-4096-a3f7-f1b11d982a49.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Support more programming languages for workspace index"
```

Comment:
This is an opportunity to link to relevant docs pages or marketing landing pages (markdown is supported), if possible.

Reply:
We do not have relevant doc pages for this.